.travis/before_install.sh

```diff
@@ -40,7 +40,7 @@ cd /tmp || die
     sudo apt-get update || die
 
     [ $BUILD_TARGET != posix-distcheck -a $BUILD_TARGET != posix-32-bit -a $BUILD_TARGET != posix-mtd -a $BUILD_TARGET != posix-ncp ] || {
-        pip install --upgrade pip || die
+        sudo pip install --upgrade pip || die
```

Comment:
In general, I think using `--user` is preferred over `sudo`.

Reply:
It's a `pip` option. right? Changed it in new pushed commit.

Reply:
Yes, a `pip` option.